### PR TITLE
Unit test case for basicget.js

### PR DESCRIPTION
**Pull request by tanaysingh3484**
Created a Unit-test file for the `basicget.js` file, using the `Chai` assertion library.

### CODE REVIEW

File: Node.js/test/basicget.test.js [hunk 1]
```diff
@@ -0,0 +1,234 @@
+const { buildMQDetails, ccdtCheck, initialise, connx, open, close, disconnect, getMessage } = require('../basicget');
```
Comment (chughts) [inline Node.js/test/basicget.test.js:1]: README needs updating with instructions on how to run the tests.

Comment (chughts) [reply Node.js/test/basicget.test.js:1]: package.json will need updating to allow for `npm test` command

File: Node.js/test/basicget.test.js [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+        msgObject = {
+            "Key": "0"
+        }        
+        msg = JSON.stringify(msgObject);
+
+    });
+
+    it('Should perform MQGET', async () => {
+        for (let i = 0; i < envConfigLength; i++) {
+            await buildMQDetails(MQDetails, credentials, i);
```
Comment (chughts) [inline Node.js/test/basicget.test.js:212]: This is a copy and paste of the put logic. Instead, just run `basicput`, then test that the get functions run.

**Review by chughts — CHANGES_REQUESTED** [review]

**Comment by tanaysingh3484** [comment]
- Updated the `MQGET` test by adding an `exec` statement to first execute the `basicput.js` file, following which the `getMessage` function is called
- Added a `README` for the test directory, which contains the information needed to run these testcases
- Updated the `basicget.js` with the required `module.exports`.

**Review by chughts — APPROVED** [review]
Seems odd to export the functions, just so they can be tested, but I guess this is a result of having a single module node.js app.

**Pull request merged**